README.md

```diff
@@ -244,6 +244,22 @@ POWER TOYS:
       --oa                 browse all search results immediately
       --replace old new    replace old tag with new tag everywhere
                            delete old tag, if new tag not specified
+      --url-redirect       when fetching an URL, use the resulting
```

Comment:
Please document these in the readme and add to the auto-completion scripts.

Reply:
This _is_ the readme though… :thinking: do you mean adding examples or something?